modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/frontend/taglib/servlet/taglib/OrganizationScreenNavigationEntry.java

```diff
@@ -119,14 +121,18 @@ public void render(HttpServletRequest request, HttpServletResponse response)
 			}
 		}
 
-		displayContext.setOrganization(organization);
-		displayContext.setOrganizationId(organizationId);
-		displayContext.setScreenNavigationCategoryKey(_categoryKey);
-		displayContext.setScreenNavigationEntryKey(_entryKey);
+		organizationScreenNavigationDisplayContext.setOrganization(
+			organization);
+		organizationScreenNavigationDisplayContext.setOrganizationId(
+			organizationId);
+		organizationScreenNavigationDisplayContext.
+			setScreenNavigationCategoryKey(_categoryKey);
+		organizationScreenNavigationDisplayContext.setScreenNavigationEntryKey(
+			_entryKey);
 
 		request.setAttribute(
 			UsersAdminWebKeys.ORGANIZATION_SCREEN_NAVIGATION_DISPLAY_CONTEXT,
-			displayContext);
+			organizationScreenNavigationDisplayContext);
```

Comment:
Thanks for taking care of this.  I remember you pointed it out during our initial review but I forgot to change it.